adding Phase 3 edits - ChandlerSamuels

chandlersamuels opened the pull request:
I added a functionality to let the user only answer each category once. I also added a print statement that shows how many questions you got wrong.

Commit:
adding Phase 3 edits - ChandlerSamuels

brandonnstone commented:
Hey man cool thanks for contributing. Did you happen to fork or did you just push it?

> On Apr 29, 2016, at 4:53 PM, Chandler Samuels notifications@github.com wrote:
> 
> I added a functionality to let the user only answer each category once. I also added a print statement that shows how many questions you got wrong.
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/bstoriz/storiz_stone_castanon_CSCI2270_FinalProject/pull/3
> 
> Commit Summary
> 
> adding Phase 3 edits - ChandlerSamuels
> File Changes
> 
> M main.cpp (172)
> M quizShow.cpp (10)
> Patch Links:
> 
> https://github.com/bstoriz/storiz_stone_castanon_CSCI2270_FinalProject/pull/3.patch
> https://github.com/bstoriz/storiz_stone_castanon_CSCI2270_FinalProject/pull/3.diff
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub

chandlersamuels commented:
Hey man,

I did fork it before I cloned the repository to my laptop, so you should be
good. I hope I did it right so that I didn't just over write your code. I'm
still pretty new to github.

On Friday, April 29, 2016, Brandon Stone notifications@github.com wrote:

> Hey man cool thanks for contributing. Did you happen to fork or did you
> just push it?
> 
> > On Apr 29, 2016, at 4:53 PM, Chandler Samuels <notifications@github.com
> > <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
> > 
> > I added a functionality to let the user only answer each category once.
> > I also added a print statement that shows how many questions you got wrong.
> > 
> > You can view, comment on, or merge this pull request online at:
> > 
> > https://github.com/bstoriz/storiz_stone_castanon_CSCI2270_FinalProject/pull/3
> > 
> > Commit Summary
> > 
> > adding Phase 3 edits - ChandlerSamuels
> > File Changes
> > 
> > M main.cpp (172)
> > M quizShow.cpp (10)
> > Patch Links:
> > 
> > https://github.com/bstoriz/storiz_stone_castanon_CSCI2270_FinalProject/pull/3.patch
> > 
> > https://github.com/bstoriz/storiz_stone_castanon_CSCI2270_FinalProject/pull/3.diff
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly or view it on GitHub
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/bstoriz/storiz_stone_castanon_CSCI2270_FinalProject/pull/3#issuecomment-215902491

brandonnstone commented:
All good either way just checking. Good luck on finals.

On Fri, Apr 29, 2016 at 5:08 PM, Chandler Samuels notifications@github.com
wrote:

> Hey man,
> 
> I did fork it before I cloned the repository to my laptop, so you should be
> good. I hope I did it right so that I didn't just over write your code. I'm
> still pretty new to github.
> 
> On Friday, April 29, 2016, Brandon Stone notifications@github.com wrote:
> 
> > Hey man cool thanks for contributing. Did you happen to fork or did you
> > just push it?
> > 
> > > On Apr 29, 2016, at 4:53 PM, Chandler Samuels <
> > > notifications@github.com
> > > <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
> > > 
> > > I added a functionality to let the user only answer each category once.
> > > I also added a print statement that shows how many questions you got
> > > wrong.
> > > 
> > > You can view, comment on, or merge this pull request online at:
> > 
> > https://github.com/bstoriz/storiz_stone_castanon_CSCI2270_FinalProject/pull/3
> > 
> > > Commit Summary
> > > 
> > > adding Phase 3 edits - ChandlerSamuels
> > > File Changes
> > > 
> > > M main.cpp (172)
> > > M quizShow.cpp (10)
> > > Patch Links:
> > 
> > https://github.com/bstoriz/storiz_stone_castanon_CSCI2270_FinalProject/pull/3.patch
> > 
> > 
> > https://github.com/bstoriz/storiz_stone_castanon_CSCI2270_FinalProject/pull/3.diff
> > 
> > > —
> > > You are receiving this because you are subscribed to this thread.
> > > Reply to this email directly or view it on GitHub
> > 
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly or view it on GitHub
> > <
> > https://github.com/bstoriz/storiz_stone_castanon_CSCI2270_FinalProject/pull/3#issuecomment-215902491
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> https://github.com/bstoriz/storiz_stone_castanon_CSCI2270_FinalProject/pull/3#issuecomment-215904248

chandlersamuels commented:
Thanks you too

On Friday, April 29, 2016, Brandon Stone notifications@github.com wrote:

> All good either way just checking. Good luck on finals.
> 
> On Fri, Apr 29, 2016 at 5:08 PM, Chandler Samuels <
> notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>>
> wrote:
> 
> > Hey man,
> > 
> > I did fork it before I cloned the repository to my laptop, so you should
> > be
> > good. I hope I did it right so that I didn't just over write your code.
> > I'm
> > still pretty new to github.
> > 
> > On Friday, April 29, 2016, Brandon Stone <notifications@github.com
> > <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
> > 
> > > Hey man cool thanks for contributing. Did you happen to fork or did you
> > > just push it?
> > > 
> > > > On Apr 29, 2016, at 4:53 PM, Chandler Samuels <
> > > > notifications@github.com
> > > > <javascript:_e(%7B%7D,'cvml','notifications@github.com');>
> > > > <javascript:_e(%7B%7D,'cvml','notifications@github.com
> > > > <javascript:_e(%7B%7D,'cvml','notifications@github.com');>');>> wrote:
> > > > 
> > > > I added a functionality to let the user only answer each category
> > > > once.
> > > > I also added a print statement that shows how many questions you got
> > > > wrong.
> > > > 
> > > > You can view, comment on, or merge this pull request online at:
> > 
> > https://github.com/bstoriz/storiz_stone_castanon_CSCI2270_FinalProject/pull/3
> > 
> > > > Commit Summary
> > > > 
> > > > adding Phase 3 edits - ChandlerSamuels
> > > > File Changes
> > > > 
> > > > M main.cpp (172)
> > > > M quizShow.cpp (10)
> > > > Patch Links:
> > 
> > https://github.com/bstoriz/storiz_stone_castanon_CSCI2270_FinalProject/pull/3.patch
> > 
> > 
> > https://github.com/bstoriz/storiz_stone_castanon_CSCI2270_FinalProject/pull/3.diff
> > 
> > > > —
> > > > You are receiving this because you are subscribed to this thread.
> > > > Reply to this email directly or view it on GitHub
> > > 
> > > —
> > > You are receiving this because you authored the thread.
> > > Reply to this email directly or view it on GitHub
> > > <
> > 
> > https://github.com/bstoriz/storiz_stone_castanon_CSCI2270_FinalProject/pull/3#issuecomment-215902491
> > 
> > 
> > —
> > You are receiving this because you commented.
> > Reply to this email directly or view it on GitHub
> > <
> > https://github.com/bstoriz/storiz_stone_castanon_CSCI2270_FinalProject/pull/3#issuecomment-215904248
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/bstoriz/storiz_stone_castanon_CSCI2270_FinalProject/pull/3#issuecomment-215905221
